test/wct-sauce.conf.json

```diff
@@ -7,7 +7,7 @@
   ],
   "clientOptions": {
     "mochaOptions": {
-      "timeout": 90000
+      "timeout": 180000
```

Comment:
This actually didn't help as I recall, so we could probably revert it. Up to you.

Reply:
It's arbitrarily high either way. No harm in keeping it this high.